lib/route.coffee

```diff
@@ -207,8 +207,15 @@ class @Route
     # Send response
     endpointContext.response.writeHead statusCode, headers
     endpointContext.response.write body
-    endpointContext.response.end()
-
+    switch statusCode
```

Comment:
Can we change this switch statement to a simple if-else, since we don't have any further cases to detect? So this could be
```coffeescript
if statusCode in [401, 403]
  # Delay response
else
  # Respond immediately
```
Seems a bit more straightforward to me.